src/interpreter/environment.cpp

```diff
@@ -4,7 +4,7 @@
 
 #include "garbage_collector.hpp"
 
-namespace Type {
+namespace Interpreter {
```

Comment:
This here is some legacy mess, I regret putting `Environment` in the `Type` namespace every day